src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -43,6 +42,16 @@ public DiagnosticsTelemetryModule()
         }
 
         /// <summary>
+        /// Gets or sets the delay between heartbeat emmissions in milliseconds.
+        /// </summary>
+        public int MillisecondsBetweenHeartbeats { get; set; }
+
+        /// <summary>
+        /// Gets or sets property names that are allowed to be sent along with the health heartbeats emitted. * means allow all default properties through.
+        /// </summary>
+        public string AllowedHeartbeatProperties { get; set; }
```

Comment:
Do you want to whitelist? It may be good idea to blacklist instead to allow disabling a subset of properties and allow all you know nothing about

Reply:
Agreed. Blacklist is superior. Done.